modules/datadict/php/datadict.class.inc

```diff
@@ -58,8 +58,26 @@ class Datadict extends \DataFrameworkMenu
      */
     public function getFieldOptions() : array
     {
+        $instruments     = \Utility::getAllInstruments();
+        $dictInstruments = array();
+        foreach ($instruments as $instrument => $name) {
+            // Since the testname does not always match the table name in the
+            // database we need to instantiate the object to get the table name
+            $iObj = \NDB_BVL_Instrument::factory(
+                $instrument,
+                '',
+                ''
+            );
+            $key  = $iObj->usesJSONData() ? $iObj->testName:$iObj->table;
+            // Append other testnames if multiple tests use the same table
+            $otherNames = '';
+            if (isset($dictInstruments[$key])) {
+                $otherNames = "/".$dictInstruments[$key];
+            }
+            $dictInstruments[$key] = $name . $otherNames;
```

Comment:
Small detail, but since `$name` refers to the last test name, can we reverse the order here for which name is displayed first ?

<img width="1438" alt="Screen Shot 2021-01-06 at 12 16 22 PM" src="https://user-images.githubusercontent.com/34260251/103799727-7c046780-5019-11eb-87e9-84d850322f43.png">